packages/celotool/src/lib/blockscout.ts

```diff
@@ -190,17 +190,26 @@ spec:
     http:
       paths:
       - path: /api/v1/(decompiled_smart_contract|verified_smart_contracts)
+        pathType: ImplementationSpecific
         backend:
-          serviceName: ${ingressName}-web
-          servicePort: 4000
+          service:
+          name: ${ingressName}-web
+          port:
+            number: 4000
       - path: /(graphql|graphiql|api)
+        pathType: ImplementationSpecific
         backend:
-          serviceName: ${ingressName}-api
-          servicePort: 4000
-      - backend:
-          serviceName: ${ingressName}-web
-          servicePort: 4000
-        path: /
+          service:
+          name: ${ingressName}-api
+          port:
+            number: 4000
+      - path: /
+        pathType: ImplementationSpecific
+        backend:
+          service:
+          name: ${ingressName}-web
+          port:
+            number: 4000
```

Comment:
I believe `name` and `port` need indenting to be below `service`?

Reply:
Good catch